<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog3</class>
 <widget class="QDialog" name="Dialog3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #ADD8E6; /* Light Blue color code */
    color: white;
}</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>291</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
    background-color: #f5f5f5;
    border: 1px solid #ddd;
    border-radius: 5px;
}

QTableWidget::item {
    color: #333;
    background-color: #fff;
    border: none;
    font-size: 500px; /* Adjust the font size as needed */
}

QTableWidget::item:selected {
    background-color: #ffcc00;
    color: #000;
}

QTableWidget::horizontalHeader {
    background-color: #007BFF;
    color: white;
    border: none;
    border-top-left-radius: 5px;
    border-top-right-radius: 5px;
}

QTableWidget::horizontalHeader::section {
    padding: 5px;
}

QTableWidget::verticalHeader {
    background-color: #007BFF;
    color: white;
    border: none;
    border-top-left-radius: 5px;
    border-bottom-left-radius: 5px;
}

QTableWidget::verticalHeader::section {
    padding: 5px;
}

QTableWidget::corner {
    background-color: #007BFF;
    border: none;
    border-top-left-radius: 5px;
}
</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>130</y>
     <width>411</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget{
    font-size: 20px; /* Set the font size */
    font-weight: bold; /* Make the text bold */
    color: white; /* Text color (white) for good contrast on a blue background */
    background-color: #3498db; /* Background color (blue) */
    padding: 10px; /* Add padding for better spacing */
   text-align: center;
}
</string>
   </property>
   <item>
    <property name="text">
     <string>1.Беліченко Юрій</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2.Бицка Тарас</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3.Боднар Артем</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4.Коваль Назар</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5.Кутник Володимир</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6.Баб'як Дмитро</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7.Курівчак Денис</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8.Подолянюк Сергій</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>440</y>
     <width>201</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/photo/photos/1316344-200 – копія.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>620</y>
     <width>891</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 18px; /* Set the font size */
    font-weight: bold; /* Make the text bold */
    color: white; /* Text color (white) for good contrast on a blue background */
    background-color: #3498db; /* Background color (blue) */
    padding: 10px; /* Add padding for better spacing */
}</string>
   </property>
   <property name="text">
    <string>Натисніть на прізвище спортсмена, щоб переглянути додаткову інформацію про нього!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    font-size: 16px; /* Set the font size */
    padding: 5px; /* Add some padding for better spacing */
    border: 2px solid #3498db; /* Border with a shade of blue color */
    border-radius: 5px; /* Rounded corners */
    background-color: #f2f2f2; /* Background color */
    color: #333; /* Text color */
}

QLineEdit:hover {
    border-color: #FFA500; /* Change border color on hover (orange) */
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    font-size: 16px; /* Set the font size */
    padding: 5px; /* Add some padding for better spacing */
    border: 2px solid #3498db; /* Border with a shade of blue color */
    border-radius: 5px; /* Rounded corners */
    background-color: #f2f2f2; /* Background color */
    color: #333; /* Text color */
}

QLineEdit:hover {
    border-color: #FFA500; /* Change border color on hover (orange) */
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 15px; /* Set the font size */
    font-weight: bold; /* Make the text bold */
    color: #FFFFFF; /* Text color (white) for good contrast on the background */
    text-shadow: 1px 1px 2px #000000; /* Add a subtle text shadow */
    background-color: #FF5733; /* Background color (a shade of orange) */
    border: 1px solid #FF4500; /* Add a border for emphasis */
    padding: 10px; /* Add some padding for better spacing */
    border-radius: 5px; /* Optional for rounded corners */
}</string>
   </property>
   <property name="text">
    <string>Введіть прізвище</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 15px; /* Set the font size */
    font-weight: bold; /* Make the text bold */
    color: #FFFFFF; /* Text color (white) for good contrast on the background */
    text-shadow: 1px 1px 2px #000000; /* Add a subtle text shadow */
    background-color: #FF5733; /* Background color (a shade of orange) */
    border: 1px solid #FF4500; /* Add a border for emphasis */
    padding: 10px; /* Add some padding for better spacing */
    border-radius: 5px; /* Optional for rounded corners */
}</string>
   </property>
   <property name="text">
    <string>Введіть ім'я</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    font-size: 16px; /* Set the font size */
    padding: 5px; /* Add some padding for better spacing */
    border: 2px solid #3498db; /* Border with a shade of blue color */
    border-radius: 5px; /* Rounded corners */
    background-color: #f2f2f2; /* Background color */
    color: #333; /* Text color */
}

QLineEdit:hover {
    border-color: #FFA500; /* Change border color on hover (orange) */
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 15px; /* Set the font size */
    font-weight: bold; /* Make the text bold */
    color: #FFFFFF; /* Text color (white) for good contrast on the background */
    text-shadow: 1px 1px 2px #000000; /* Add a subtle text shadow */
    background-color: #FF5733; /* Background color (a shade of orange) */
    border: 1px solid #FF4500; /* Add a border for emphasis */
    padding: 10px; /* Add some padding for better spacing */
    border-radius: 5px; /* Optional for rounded corners */
}</string>
   </property>
   <property name="text">
    <string>Введіть розряд</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font-size: 15px; /* Set the font size */
    padding: 10px 20px; /* Add padding for button size */
    background-color: #FF5733; /* Button background color (a shade of orange) */
    color: #FFFFFF; /* Text color (white) */
    border: 2px solid #FF4500; /* Button border color (a shade of orange) */
    border-radius: 5px; /* Rounded button corners */
}

QPushButton:hover {
    background-color: #800080; /* Deep purple background on hover */
    border: 2px solid #800080; /* Change border color to deep purple on hover */
}

QPushButton:pressed {
    background-color: #3498db; /* Change background color when pressed (blue) */
    border: 2px solid #1E6D88; /* Change border color when pressed */
}
</string>
   </property>
   <property name="text">
    <string>Внести в таблицю</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font-size: 15px; /* Set the font size */
    padding: 10px 20px; /* Add padding for button size */
    background-color: #FF5733; /* Button background color (a shade of orange) */
    color: #FFFFFF; /* Text color (white) */
    border: 2px solid #FF4500; /* Button border color (a shade of orange) */
    border-radius: 5px; /* Rounded button corners */
}

QPushButton:hover {
    background-color: #800080; /* Deep purple background on hover */
    border: 2px solid #800080; /* Change border color to deep purple on hover */
}

QPushButton:pressed {
    background-color: #3498db; /* Change background color when pressed (blue) */
    border: 2px solid #1E6D88; /* Change border color when pressed */
}
</string>
   </property>
   <property name="text">
    <string>Оновити таблицю</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font-size: 15px; /* Set the font size */
    padding: 10px 20px; /* Add padding for button size */
    background-color: #FF5733; /* Button background color (a shade of orange) */
    color: #FFFFFF; /* Text color (white) */
    border: 2px solid #FF4500; /* Button border color (a shade of orange) */
    border-radius: 5px; /* Rounded button corners */
}

QPushButton:hover {
    background-color: #800080; /* Deep purple background on hover */
    border: 2px solid #800080; /* Change border color to deep purple on hover */
}

QPushButton:pressed {
    background-color: #3498db; /* Change background color when pressed (blue) */
    border: 2px solid #1E6D88; /* Change border color when pressed */
}
</string>
   </property>
   <property name="text">
    <string>Видалити дані</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font-size: 15px; /* Set the font size */
    padding: 10px 20px; /* Add padding for button size */
    background-color: #FF5733; /* Button background color (a shade of orange) */
    color: #FFFFFF; /* Text color (white) */
    border: 2px solid #FF4500; /* Button border color (a shade of orange) */
    border-radius: 5px; /* Rounded button corners */
}

QPushButton:hover {
    background-color: #800080; /* Deep purple background on hover */
    border: 2px solid #800080; /* Change border color to deep purple on hover */
}

QPushButton:pressed {
    background-color: #3498db; /* Change background color when pressed (blue) */
    border: 2px solid #1E6D88; /* Change border color when pressed */
}
</string>
   </property>
   <property name="text">
    <string>Оновити дані</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
